### DISCUSSION TITLE
[Unstable] Acid emitters spawned by the acid grenade will make loud noise when hit the structure.

### DISCUSSION THREAD

**Discussion by zhu-rengong** [FakeFishGames/Barotrauma#14229] [Bug reports]
### Disclaimers

- [X] I have searched the issue tracker to check if the issue has already been reported.
- [ ] My issue happened while using mods.

### What happened?

As the title saying, I think it's unreal.

### Reproduction steps

1. Detonator a acid grenade indoor
2. You will constantly hear the walls being hit hard

### Bug prevalence

Happens every now and then

### Single player or multiplayer?

Multiplayer hosted from the in-game menu (= using a listen server)

### Version

v1.6.100.0

### Which operating system did you encounter this bug on?

Windows

**Comment by Jasontti** [comment]
Tested the fix for this.
Emitters don't cause noise when used in hulls
Closing discussion as resolved
